hilti/toolchain/CMakeLists.txt

```diff
@@ -191,11 +191,11 @@ set_config_val(HILTI_CONFIG_TOOLCHAIN_CXX_LIBRARY_DIRS
 # LD flags
 set_config_val(
     HILTI_CONFIG_RUNTIME_LD_FLAGS_DEBUG
-    "${EXTRA_CXX_FLAGS} ${CMAKE_CXX_FLAGS} ${addl_ld_flags} ${EXTRA_LD_FLAGS} ${CMAKE_EXE_LINKER_FLAGS_INIT} -Wno-unused-command-line-argument"
+    "${EXTRA_CXX_FLAGS} ${CMAKE_CXX_FLAGS} ${addl_ld_flags} ${EXTRA_LD_FLAGS} ${CMAKE_EXE_LINKER_FLAGS_INIT} -Wno-unused-command-line-argument -Wno-invalid-offsetof"
 )
 set_config_val(
     HILTI_CONFIG_RUNTIME_LD_FLAGS_RELEASE
-    "${EXTRA_CXX_FLAGS} ${CMAKE_CXX_FLAGS} ${addl_ld_flags} ${EXTRA_LD_FLAGS} ${CMAKE_EXE_LINKER_FLAGS_INIT} -Wno-unused-command-line-argument"
+    "${EXTRA_CXX_FLAGS} ${CMAKE_CXX_FLAGS} ${addl_ld_flags} ${EXTRA_LD_FLAGS} ${CMAKE_EXE_LINKER_FLAGS_INIT} -Wno-unused-command-line-argument -Wno-invalid-offsetof"
```

Comment:
Oh, that is a left-over from me: Benjamin suggested to move the warning  suppression into `config.cc.in` but I never removed it here.

Reply:
```suggestion
    "${EXTRA_CXX_FLAGS} ${CMAKE_CXX_FLAGS} ${addl_ld_flags} ${EXTRA_LD_FLAGS} ${CMAKE_EXE_LINKER_FLAGS_INIT}"
```

Reply:
Makes sense, I had been wondering about the duplication.